docs/api/covidcast-signals/_source-template.md

```diff
@@ -72,3 +72,30 @@ backfill here.
 If the signal has specific licensing or sourcing that should be acknowledged,
 describe it here. Also, include links to source websites for data that is
 scraped or received from another source.
+
+## Licensing
+
+### For signals constructed by Delphi:
+
+This data is provided under the terms of the [Creative Commons Attribution
+license](https://creativecommons.org/licenses/by/4.0/). If you use this data,
+you must cite [Delphi](https://delphi.cmu.edu/) COVIDcast API, preferably by
+providing a link to this page.
```

Comment:
Tiny nit around linking: with Delphi being linked earlier in this sentence, it initially reads like "you need to link back to the delphi website". But if you keep reading, then you realize that the Delphi website is incidental and really "you need to link to the current page". It might be more straightforward to remove the link to Delphi.

Reply:
I think I changed my mind about linking halfway through writing that. Do you think linking to the current page is the best idea? It seemed sound to me (so you're always providing a link to the detailed information about the data), but maybe if we publish a paper about the API we can change the text to point there.

Reply:
Optimal approach would be to just provide boilerplate citation text.

Reply:
How about this?

> If you use this data, you must cite Delphi's COVIDcast API. For example:
>
> > Data from the COVIDcast API, by the Delphi group at Carnegie Mellon University. <https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/SOURCENAME.html>

Reply:
Everything seems pretty straight forward to me. Adding the example helps clarify at least one way of citing the use of Covidcast data.

Reply:
It feels again like a divergence is happening, and I'd like to understand better.

In this case, the API has citation text in the top-level [readme](https://github.com/cmu-delphi/delphi-epidata#citing), which is linked to by the COVIDcast [docs](https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html#covidcast-epidata-api). Can you clarify why a different citation would be desired here? Hypothetically, if someone uses a `covidcast` signal, would we want them to cite Epidata and COVIDcast separately?

More thoughts to come.

Reply:
Some stray thoughts around terminology and citations.

I'd like to see unified terminology around the api and covidcast, and I think that means replacing "COVIDcast API" with "Epidata API".

COVIDcast encompasses many efforts, such as: the interactive website, acquisition/curation/generation of various signals, the `covidcast`-specific client libraries, the blog (some posts at least), etc. But I wouldn't consider the API to be a COVIDcast project.

However, I think it's important to recognize the huge contributions made by a large number of people to the various COVIDcast efforts. So I don't think it's optimal to cite the Epidata API as a proxy for the separate body of COVIDcast work. Where the two projects touch — the `covidcast` endpoint of the Epidata API — I think maybe a dual citation is not a terrible thing?

So for example, if there's a signal that Delphi generates (i.e. we're not just mirroring someone else) — like the indicator combination — then maybe we ought to have users cite the Epidata API for hosting that signal, and also cite the "COVIDcast Project" to satisfy the signal's CCBY license?

Which leads to another point, which is that we should have a COVIDcast Project page, with a list of contributors and a copy-pasta citation. I _think_ this is in the works as part of the site redesign.

Reply:
We do have [a public list of contributors](https://covidcast.cmu.edu/covid19-response-team.html). It was initially the "Delphi COVID-19 Response Team", but a few months ago it was decided that everyone is just part of Delphi now, and there is no separate COVID-19 team. I believe the plan is for this to be merged into a general list of Delphi members on the new site (but I might be wrong).

The practical concerns are that

* we already named the API clients "covidcast"
* we refer to the "COVIDcast API" in various presentations and published posts/docs, and changing the name will cause confusion
* people can download data without using the API at all, via the CSV download feature on the COVIDcast site. [Technically the API is still involved, but the user doesn't see it; they just know they're on COVIDcast]
* people have a hard enough time consistently giving credit with one source to cite; having two citations invites confusion

I'm not sure how to balance these and pick one option. But we clearly need a solution soonish so people can cite our work.

Reply:
* @capnrefsmmat : I now understand better what you meant by me being a fan of "COVIDcast EpiData API" -- you mean vs. other forms of tying the two brand names.  I had thought you meant vs. using Epidata.  My bad.
* "Delphi EpiData" is a known brand name among long-term participants in the CDC forecasting challenges, as well as CDC itself.
* I can live with either "COVIDcast EpiData API" or "COVIDcast endpoint of the EpiData API", or even using both with the former being an informal shorthand for the latter.  What I prefer to avoid is "COVIDcast API", because at some point soon will will go back to revamping and expanding the other endpoints of Epidata, and it's important for users to understand that conceptually it's all one thing.  If we retroactively replace all "COVIDcast API" with "COVIDcast EpiData API" in our blogs and presentations, I don't think anyone will be confused.
* EpiData is not just an API, it is also the database itself, including lots of flu-related signals, going back 20 years, dengue signals, norovirus  signals, etc.  At some point we may want to distinguish the API from the DB (for example if there are multiple DBs managed by different organizations, all speaking EpiData), but I guess we don't need to worry about it now.